docs/api_reference/src/quantities.tex

```diff
@@ -3917,21 +3987,222 @@
 };
 \end{codeblock}
 
-\rSec2[qty.pt.concepts]{Quantity point concepts}
+\rSec2[qty.pt.orig]{Point origin}
 
-\begin{itemdecl}
-template<typename T>
-concept @\deflibconcept{PointOrigin}@ = @\unspecnc@;
+\pnum
+This subclause specifies the components
+for defining the origin of an affine space\irefiev{102-03-02}.
+An \defn{origin} is the point O of the third property of an affine space.
```

Comment:
What does "of the third property" mean? What are the other properties? Are they listed somewhere in our specs in exactly that order?

Reply:
Here: <https://www.electropedia.org/iev/iev.nsf/display?openform&ievref=102-03-02>.
The definition of "origin" definitely needs some work.

Reply:
Do you mean that work is needed in wording or the code as well?

Reply:
In the wording only.